lib/compilers/_all.js

```diff
@@ -22,67 +22,68 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-export { AdaCompiler } from './ada';
-export { AnalysisTool } from './analysis-tool';
-export { AssemblyCompiler } from './assembly';
-export { NasmCompiler } from './nasm';
-export { Cc65Compiler } from './cc65';
-export { CircleCompiler } from './circle';
-export { ClangCompiler } from './clang';
-export { ClangCudaCompiler } from './clang';
-export { ClangHipCompiler } from './clang';
-export { CleanCompiler } from './clean';
-export { CprocCompiler } from './cproc';
-export { CrystalCompiler } from './crystal';
-export { DartCompiler } from './dart';
-export { DefaultCompiler } from './default';
-export { DMDCompiler } from './dmd';
-export { CSharpCompiler } from './dotnet';
-export { EllccCompiler } from './ellcc';
-export { ErlangCompiler } from './erlang';
-export { EWARMCompiler } from './ewarm';
-export { EWAVRCompiler } from './ewavr';
-export { FakeCompiler } from './fake-for-test';
-export { FSharpCompiler } from './dotnet';
-export { FortranCompiler } from './fortran';
-export { GCCCompiler } from './gcc';
-export { GCCRSCompiler } from './gccrs';
-export { GolangCompiler } from './golang';
-export { HaskellCompiler } from './haskell';
-export { ISPCCompiler } from './ispc';
-export { JavaCompiler } from './java';
-export { KotlinCompiler } from './kotlin';
-export { LDCCompiler } from './ldc';
-export { LLCCompiler } from './llc';
-export { LLVMmcaTool } from './llvm-mca';
-export { NimCompiler } from './nim';
-export { NvccCompiler } from './nvcc';
-export { OCamlCompiler } from './ocaml';
-export { OptCompiler } from './opt';
-export { OSACATool } from './osaca';
-export { FPCCompiler } from './pascal';
-export { PascalWinCompiler } from './pascal-win';
-export { PPCICompiler } from './ppci';
-export { PtxAssembler } from './ptxas';
-export { PythonCompiler } from './python';
-export { RubyCompiler } from './ruby';
-export { RustCompiler } from './rust';
-export { RustcCgGCCCompiler } from './rustc-cg-gcc';
-export { MrustcCompiler } from './mrustc';
-export { ScalaCompiler } from './scala';
-export { SdccCompiler } from './sdcc';
-export { SolidityCompiler } from './solidity';
-export { SPIRVCompiler } from './spirv';
-export { SwiftCompiler } from './swift';
-export { TenDRACompiler } from './tendra';
-export { TinyCCompiler } from './tinyc';
-export { TypeScriptCompiler } from './typescript';
-export { VBCompiler } from './dotnet';
-export { Win32Compiler } from './win32';
-export { Win32VcCompiler } from './win32-vc';
-export { Win32Vc6Compiler } from './win32-vc6';
-export { WineVcCompiler } from './wine-vc';
-export { WslVcCompiler } from './wsl-vc';
-export { ZigCompiler } from './zig';
-export { ZigCC } from './zigcc';
-export { ZigCXX } from './zigcxx';
+export {AdaCompiler} from './ada';
```

Comment:
Ah autoformat :) We really need to "just" autoformat everything. Maybe I'll look at that tonight, we have an historic low number of PRs out